src/host/tracing.cpp

```diff
@@ -366,6 +389,7 @@ void Tracing::s_TraceInputRecord(const INPUT_RECORD& inputRecord)
             "Focus Event Input Record",
             TraceLoggingBool(inputRecord.Event.FocusEvent.bSetFocus, "bSetFocus"),
             TraceLoggingLevel(WINEVENT_LEVEL_VERBOSE),
+            TraceLoggingKeyword(TIL_KEYWORD_TRACE),
             TraceLoggingKeyword(TraceKeywords::Input));
         break;
     default:

```

Comment:
I can't mark this on GitHub, but this default case doesn't have the new `TraceLoggingKeyword` argument.

---

`Tracing::TraceFailure` is missing it as well.

Reply:
Got em both